app/lib/registries/organisations_registry.rb

```diff
@@ -15,6 +15,10 @@ def organisations
       {}
     end
 
+    def values
```

Comment:
Might be worth renaming the organisations method to values (and in other registries).

Reply:
I will raise another PR for this (there will be quite a few changes) but also to rename the method from `values` to `all` and then use this to make `allowed_values_from_registry` clearer.